### PR TITLE
Highlight the search string in company result list

### PR DESCRIPTION
**Pull request by teneightfive**
This is a first pass at highlighting the text that has been searched for in the companies results list.

### CODE REVIEW

File: controllers/SearchController.js [hunk 1]
```diff
@@ -12,6 +12,15 @@ function handleError(res, error) {
   });
 }
 
+function highlightText(str, searchTerm) {
```
Comment (ztolley) [inline controllers/SearchController.js]: seems to call to lower case multiple times and if there is nothing to replace it returns the original mixed case but if there is a match it forces it to lower case.

File: stylesheets/_results-list.scss [hunk 2]
```diff
@@ -11,6 +11,7 @@
       font-weight: 400;
       margin: 0 0 em(5);
       padding: 0;
+      text-transform: capitalize;
```
Comment (ztolley) [inline stylesheets/_results-list.scss]: why?

Comment (teneightfive) [inline stylesheets/_results-list.scss]: Was a quick way of standardising the response from companies house. Some company names come back all uppercase, some don't

File: controllers/SearchController.js (hunk 1 above)
Comment (teneightfive) [inline controllers/SearchController.js]: Have improved it

**Pull request merged**